src/Core/KeyManagement/UserKey/Implementations/RotateUserKeyCommand.cs

```diff
@@ -1,13 +1,13 @@
-﻿using Bit.Core.Auth.Models.Data;
-using Bit.Core.Auth.Repositories;
+﻿using Bit.Core.Auth.Repositories;
 using Bit.Core.Entities;
+using Bit.Core.KeyManagement.Models.Data;
 using Bit.Core.Repositories;
 using Bit.Core.Services;
 using Bit.Core.Tools.Repositories;
 using Bit.Core.Vault.Repositories;
 using Microsoft.AspNetCore.Identity;
 
-namespace Bit.Core.Auth.UserFeatures.UserKey.Implementations;
+namespace Bit.Core.KeyManagement.UserFeatures.UserKey.Implementations;
```

Comment:
wrong namespace to the file location, should be:
```suggestion
namespace Bit.Core.KeyManagement.UserKey.Implementations;
```
Or move the file to "UserFeatures" directory.